src/modules/homepage/index.tsx

```diff
@@ -0,0 +1,430 @@
+
+        <Box display='flex' flexDirection='row' justifyContent='space-between'>
+          <Box
+            width='164px'
+            height='80px'
+            borderRadius='28px'
+            bgcolor='#00A86B'
+            marginLeft='16px'
+            display='flex'
+            flexDirection='row'
+            alignItems='center'
+          >
+            <Box
+              width='44px'
+              height='44px'
+              borderRadius='16px'
+              bgcolor='#fff'
+              color='#00A86B'
+              display='flex'
+              flexDirection='column'
+              alignItems='center'
+              justifyContent='center'
+              marginLeft='16px'
+            >
+              <ArrowDownwardIcon />
+              <CameraAltIcon />
+            </Box>
+            <Box color='#fff' marginLeft='8px'>
+              <Typography sx={{ fontSize: '14px' }}>Income</Typography>
+              <Typography sx={{ fontSize: '22px', fontWeight: '600' }}>
+                $5000
+              </Typography>
+            </Box>
+          </Box>
+          <Box
+            width='164px'
+            height='80px'
+            borderRadius='28px'
+            bgcolor='#FD3C4A'
+            marginRight='16px'
+            display='flex'
+            flexDirection='row'
+            alignItems='center'
+          >
+            <Box
+              width='44px'
+              height='44px'
+              borderRadius='16px'
+              bgcolor='#fff'
+              color='#FD3C4A'
+              display='flex'
+              flexDirection='column'
+              alignItems='center'
+              justifyContent='center'
+              marginLeft='16px'
+            >
+              <ArrowUpwardIcon />
+              <CameraAltIcon />
+            </Box>
+            <Box color='#fff' marginLeft='8px'>
+              <Typography sx={{ fontSize: '14px' }}>Expenses</Typography>
+              <Typography sx={{ fontSize: '22px', fontWeight: '600' }}>
+                $1200
+              </Typography>
+            </Box>
+          </Box>
+        </Box>
+
+        {/* <StyledPageContentDiv>
+                    <div>
+                        <Link to='/onboarding/introduction'>
+                            Onboarding introduction
+                        </Link>
+                    </div>
+                    <div>
+                        <Link to='/expense/new'>Add new expense</Link>
+                    </div>
+                </StyledPageContentDiv> */}
+      </StyledRectangle>
+
+      <Box padding='8px'>
+        <Typography
+          width='375px'
+          height='48px'
+          fontSize='18px'
+          fontWeight='600'
+          variant='h6'
+          lineHeight='21.78px'
+          textAlign='left'
+        >
+          Spend Frequency
+        </Typography>
+      </Box>
+
+      <Box width='375px' height='185.5px'>
+        <Graphic />
+      </Box>
+
+      <Box display='flex' flexDirection='row'>
```

Comment:
This should be an element on it's own.
Use Stack instead of Box.

Reply:
You'd need to create the first variation from the Tabs shared components:
https://www.figma.com/design/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?node-id=642-11&t=owVDwKh8pLVF3EUv-0

Reply:
I used stack instead of Box and ı created variation but ı am not sure if ı am correct here !